htdocs/core/class/html.formcompany.class.php

```diff
@@ -1032,47 +1032,90 @@ public function selectProspectCustomerType($selected, $htmlname = 'client', $htm
 			return '';
 		}
 
-		$out = '<select class="flat ' . $morecss . '" name="' . $htmlname . '" id="' . $htmlidname . '">';
-		if ($typeinput == 'form') {
-			if ($allowempty || ($selected == '' || $selected == '-1')) {
-				$out .= '<option value="-1">';
-				if (is_numeric($allowempty)) {
-					$out .= '&nbsp;';
-				} else {
-					$out .= $langs->trans($allowempty);
+		if (!$conf->customizeprospectcustomerfield->enabled) {
```

Comment:
@thomasjulie add a hook to be able to personalize

Reply:
You can take a look at this page to know how to
https://wiki.dolibarr.org/index.php/Hooks_system#Adding_a_hook_point_into_code